controller/tenants.go

```diff
@@ -73,3 +94,57 @@ func (c *TenantsController) Search(ctx *app.SearchTenantsContext) error {
 	}
 	return ctx.OK(&result)
 }
+
+// Delete runs the `delete` action to deprovision a user
+func (c *TenantsController) Delete(ctx *app.DeleteTenantsContext) error {
+	if !token.IsSpecificServiceAccount(ctx, "fabric8-auth") {
+		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError("Wrong token"))
+	}
+	tenantID := ctx.TenantID
+	// fetch the cluster the user belongs to
+	usr, err := c.userService.GetUser(ctx, tenantID)
```

Comment:
Not really needed. We have that from our own DB

Reply:
ok, fixed in 764fc32af4f0ecb16553db2bfe4b5a9101d018d6